app/src/androidTest/java/com/android/voyageur/ui/overview/OverviewScreenTest.kt

```diff
@@ -3,24 +3,33 @@ package com.android.voyageur.ui.overview
 import androidx.compose.ui.test.assertIsDisplayed
 import androidx.compose.ui.test.junit4.createComposeRule
 import androidx.compose.ui.test.onNodeWithTag
+import androidx.compose.ui.test.performClick
+import com.android.voyageur.model.trip.TripRepository
+import com.android.voyageur.model.trip.TripsViewModel
 import com.android.voyageur.ui.navigation.NavigationActions
 import com.android.voyageur.ui.navigation.Route
+import com.android.voyageur.ui.navigation.Screen
 import org.junit.Before
 import org.junit.Rule
 import org.junit.Test
 import org.mockito.Mockito.mock
 import org.mockito.Mockito.`when`
+import org.mockito.kotlin.verify
 
 class OverviewScreenTest {
+  private lateinit var tripRepository: TripRepository
   private lateinit var navigationActions: NavigationActions
+  private lateinit var tripViewModel: TripsViewModel
 
   @get:Rule val composeTestRule = createComposeRule()
 
   @Before
   fun setUp() {
+    tripRepository = mock(TripRepository::class.java)
     navigationActions = mock(NavigationActions::class.java)
+    tripViewModel = TripsViewModel(tripRepository)
     `when`(navigationActions.currentRoute()).thenReturn(Route.OVERVIEW)
-    composeTestRule.setContent { OverviewScreen(navigationActions) }
+    composeTestRule.setContent { OverviewScreen(tripViewModel, navigationActions) }
   }
 
```

Comment:
Could you also add displayTextWhenNoTrips test? There's something similar done in the bootcamp, it tests that the empty prompt is displayed when no trips were added.

Reply:
Thanks for the suggestion! I added the test, and added another one to verify it also displays card items.

Reply:
The tests look good, thanks for adding them!